### DISCUSSION TITLE
Error:The domain <domain> is already owned by another Vercel account.

### DISCUSSION THREAD

**Discussion by kenneth-ascend** [vercel/community#316] [Help] [answered]
Hello. 

I am having issues recovering ownership of my domain. I am working with someone who transferred a deployment to my account. When I try to set up the custom domain, I receive the following message: 

Error:The domain <domain> is already owned by another Vercel account.

My colleague has removed the project completely from his Vercel account. How do we remove the domain from his Vercel account completely?

**Comment by ismaelrumzan** [comment, marked as answer]
Hi there,
Please navigate to the domain settings of the personal or team account where the domain is used as explained here:
https://vercel.com/docs/concepts/projects/custom-domains#step-2:-navigating-to-domain-settings

You can then remove the domain from there.

**Reply by kenneth-ascend** [reply]
Got it. Thank you!
